CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Add set-bubble-transparent command to set the bubble transparent until the key is released (Fixes #608)
 * Deselect bottom tab when error panel is hidden
 * Add linter:lint command (Fixes #624)
+* Hide Status Bar buttons when Active Pane is not an editor
```

Comment:
You're going to need to merge master back into this branch. All the "Upcoming" changes above this were released as `1.0.5`.